internal/cephfs/clone.go

```diff
@@ -94,6 +96,9 @@ func createCloneFromSubvolume(ctx context.Context, volID, cloneID volumeID, volO
 	case cephFSCloneInprogress:
 		util.ErrorLog(ctx, "clone is in progress for %v", cloneID)
 		return ErrCloneInProgress
+	case cephFSClonePending:
+		util.ErrorLog(ctx, "clone is pending for %v", cloneID)
+		return ErrClonePending
```

Comment:
you should return in-progress here, if not we need to handle inprogress and clonepending both error cases in caller

Reply:
sounds good to add both cases i thin

Reply:
Maybe add a helper function to make it easier to backoff/retry when cloning is not complete yet.